config/v1alpha1/techpreview.imagepolicy.testsuite.yaml

```diff
@@ -0,0 +1,417 @@
+    expected: |
+      apiVersion: config.openshift.io/v1alpha1
+      kind: ImagePolicy
+      spec:
+        scopes:
+        - localhost/foo/bar
+        policy:
+          rootOfTrust:
+            policyType: PublicKey
+            publicKey:
+              keyData: Zm9vIGJhcg==
+  - name: Test scope should allow 'example.com/foo/bar'
+    initial: |
+      apiVersion: config.openshift.io/v1alpha1
+      kind: ImagePolicy
+      spec:
+        scopes:
+        - example.com/foo/bar
+        policy:
+          rootOfTrust:
+            policyType: PublicKey
+            publicKey:
+              keyData: Zm9vIGJhcg==
+    expected: |
+      apiVersion: config.openshift.io/v1alpha1
+      kind: ImagePolicy
+      spec:
+        scopes:
+        - example.com/foo/bar
+        policy:
+          rootOfTrust:
+            policyType: PublicKey
+            publicKey:
+              keyData: Zm9vIGJhcg==
+  - name: Test scope should allow tag 'example.com/foo/bar:latest'
+    initial: |
+      apiVersion: config.openshift.io/v1alpha1
+      kind: ImagePolicy
+      spec:
+        scopes:
+        - example.com/foo/bar:latest
+        policy:
+          rootOfTrust:
+            policyType: PublicKey
+            publicKey:
+              keyData: Zm9vIGJhcg==
+    expected: |
+      apiVersion: config.openshift.io/v1alpha1
+      kind: ImagePolicy
+      spec:
+        scopes:
+        - example.com/foo/bar:latest
+        policy:
+          rootOfTrust:
+            policyType: PublicKey
+            publicKey:
+              keyData: Zm9vIGJhcg==
+  - name: Test scope should allow full specification digest
+    initial: |
+      apiVersion: config.openshift.io/v1alpha1
+      kind: ImagePolicy
+      spec:
+        scopes:
+        - example.com/namespace/namespace@sha256:b7e686e30346e9ace664fa09c0275262f8b9a443ed56d22165a0e201f6488c13
+        policy:
+          rootOfTrust:
+            policyType: PublicKey
+            publicKey:
+              keyData: Zm9vIGJhcg==
+    expected: |
+      apiVersion: config.openshift.io/v1alpha1
+      kind: ImagePolicy
+      spec:
+        scopes:
+        - example.com/namespace/namespace@sha256:b7e686e30346e9ace664fa09c0275262f8b9a443ed56d22165a0e201f6488c13
+        policy:
+          rootOfTrust:
+            policyType: PublicKey
+            publicKey:
+              keyData: Zm9vIGJhcg==
+  - name: Test scope should allow '*.example.com'
+    initial: |
+      apiVersion: config.openshift.io/v1alpha1
+      kind: ImagePolicy
+      spec:
+        scopes:
+        - "*.example.com"
+        policy:
+          rootOfTrust:
+            policyType: PublicKey
+            publicKey:
+              keyData: Zm9vIGJhcg==
+    expected: |
+      apiVersion: config.openshift.io/v1alpha1
+      kind: ImagePolicy
+      spec:
+        scopes:
+        - "*.example.com"
+        policy:
+          rootOfTrust:
+            policyType: PublicKey
+            publicKey:
+              keyData: Zm9vIGJhcg==
```

Comment:
Nit, can you include a newline character at the end of the file to get rid of this red marker please